### DISCUSSION TITLE
[Enhancement] [transfer] 树形穿梭框

### DISCUSSION THREAD

**Discussion by xrhh** [element-plus/element-plus#15762] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `transfer`

## 穿梭框能否使用树形结构数据
###其它UI已有此功能
https://www.naiveui.com/zh-CN/os-theme/components/transfer

### Additional comments

_(empty)_

![image](https://github.com/element-plus/element-plus/assets/58212030/c696d2d7-fd45-4b91-8492-e59d946599d6)

**Comment by hametan** [comment]
![5c0ae6d1b819e30388dfa74e30169e62](https://github.com/element-plus/element-plus/assets/11947803/54f68279-62e3-485a-b76c-a08a8212e0fd)
补一张图。
